src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -62,7 +69,13 @@
         will be pooled.
 
     * :environment-vars - A map of environment variables and their values to be
-        passed through to the JRuby scripting container and visible to any Ruby code."
+        passed through to the JRuby scripting container and visible to any Ruby code.
+
+    * ::profiling-mode - The value to use for JRuby's ProfilerMode setting. Legal
```

Comment:
Is this supposed to have two colons in front of it, or just one like the others?

Reply:
just one. i presume i messed it up or cursive's refactoring did something wrong